test/uk/gov/hmrc/nisp/utils/BreadcrumbSpec.scala

```diff
@@ -47,6 +50,18 @@ class BreadcrumbSpec extends UnitSpec with OneAppPerSuite {
       MockBreadcrumb.generateHeaderUrl()(fakeRequestSP, nispUser) should not include "Gaps+in+your+record"
     }
 
+    "return an item text without a 'name' variable, when user.name has no value" in {
+      MockBreadcrumb.generateHeaderUrl()(fakeRequestSP, emptyNispUser) should not include ("account?name=")
+    }
+
+    "return an item text with no 'name' variable, when user.name has no value" in {
+      MockBreadcrumb.generateHeaderUrl()(fakeRequestSP, emptyNispUser) should include ("account?item_text")
```

Comment:
This test doesn't really do anything, the name could be elsewhere in the query string.
If you make the above correction this test can be removed.